core/admin/mailu/models.py

```diff
@@ -245,7 +267,7 @@ class User(Base, Email):
 
     # Filters
     forward_enabled = db.Column(db.Boolean(), nullable=False, default=False)
-    forward_destination = db.Column(db.String(255), nullable=True, default=None)
+    forward_destination = db.Column(CommaSeparatedList(), nullable=True, default=None)
```

Comment:
Reverting this back to db.String prevents the following error. It does not seem to impact the other reported errors.

```
$ docker-compose exec admin python3 manage.py admin root usrpro.io password
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1127, in _execute_context
    context = constructor(dialect, self, conn, *args)
  File "/usr/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 667, in _init_compiled
    param.append(processors[key](compiled_params[key]))
  File "/usr/lib/python3.6/site-packages/sqlalchemy/sql/type_api.py", line 1160, in process
    return process_param(value, dialect)
  File "/app/mailu/models.py", line 63, in process_bind_param
    raise TypeError("Shoud be a list")
TypeError: Shoud be a list
```